package.json

```diff
@@ -14,7 +14,7 @@
     "lint": "eslint src/"
   },
   "peerDependencies": {
-    "cypress": "^6.5.0"
+    "cypress": "^6.5.0 || ^7.0.0 || ^8.0.0"
```

Comment:
Is it compatible with latest? https://github.com/cypress-io/cypress/releases/tag/v9.1.0

Reply:
yes, we are using Cypress 9.1.0 with this plugin and it works :)